### ISSUE TITLE
The status bar will be hidden when you set android_disable_fullscreen to true, except for the splash screen.

### ISSUE THREAD

**Issue by basashi-G** [henriquearthur/flutter_native_splash#72]

**Comment by basashi-G** [comment]
The status bar will be hidden when you set android_disable_fullscreen to true, except for the splash screen, Like this.

![Screenshot_1591255189](https://user-images.githubusercontent.com/53048311/83726936-3c3e6180-a67f-11ea-8268-93a4a6a597b6.png)

**Comment by ninest** [comment]
I was also experiencing the same problem. I believe you have to write some Java/Kotlin code to fix. In the mean time, I fixed it disabling fullscreen:
```yaml
flutter_native_splash:
  image: "icons/splash.png"
  color: "111111"
  android_disable_fullscreen: true
```

Similar issue: #67

**Comment by roveroniandrea** [comment]
I had the same issue and fixed it in styles.xml (android/app/src/main/res/values)
I found this lines
```
<style name="NormalTheme" parent="@android:style/Theme.Black.NoTitleBar">
        <item name="android:windowBackground">@android:color/white</item>
        <item name="android:windowFullscreen">true</item>
 </style>
```
In particular, ```<item name="android:windowFullscreen">true</item>``` was added by the plugin, I simply deleted it and now it works. Maybe this plugin does not remove it even if fullscreen property set to false

**Comment by girish54321** [comment]
@roveroniandrea You are awesome you saved me  ;-)

**Comment by jonbhanson** [comment]
Should be resolved by 48981828e831768c4ad89a35475e8f9f24e0aca5.
